recipes/sdl_ttf/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class SdlttfConan(ConanFile):
+    name = "sdl_ttf"
+    description = "A TrueType font library for SDL"
+    license = "ZLIB"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib